README.md

```diff
@@ -160,6 +164,23 @@ You can also build with stack inside this shell as follows:
 $ stack --stack-yaml stack-nix.yaml build
 ```
 
+### Hoogle Documentation
+
+Nix has out-of-the-box Hoogle integration.  It allows you to run a local
+Hoogle server with docs for all of the project dependencies.  This is really
+nice for two reasons:
+
+1. You know you're getting correct docs for the exact versions of the packages
+   you are depending on.
+
+2. You don't have to have an internet connection to access them.
+
+To start the hoogle server, run this command from the project root directory.
+
+```
+nix-shell -p 'with import ./. {}; ghc.ghcWithHoogle (p: [p.proj])' --run 'hoogle server --local'
```

Comment:
I hit this on the latest master. Am I doing this right?

```
~/code/github/kadena-io/pact master* 
❯ nix-shell -p 'with import ./. {}; ghc.ghcWithHoogle (p: [p.proj])' --run 'hoogle server --local'
error: attribute 'proj' missing, at (string):1:137
(use '--show-trace' to show detailed location information)
```

Reply:
Oh, you have to replace `proj` with the name of the local package so that you get that package's docs included as well.  In this case that would be "pact".

Reply:
I'll add to this PR explaining that.

Reply:
Ok, added.